aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -145,6 +145,9 @@ const char* _cublasGetErrorEnum(cublasStatus_t error) {
 
 template <>
 void gemm<double>(CUDABLAS_GEMM_ARGTYPES(double)) {
+  // NOTE: If you copy and paste this alert to a different operation, you must
+  //    document the operation in torch/__init__.py:set_deterministic(), and copy
+  //    and paste this note.
```

Comment:
I think you should try to make this just one line, to reduce the likelihood that we have to do a giant multi-line sed everywhere if we need to update this documentation. Maybe `// Before modifying/copying this code, please see Note [set_deterministic]`

Reply:
Good idea. How about this, to make it a bit easier to find: `NOTE: Before modifying/copying, see aten/src/ATen/Contex.h:alertCuBLASConfigNotDeterministic`

And then for the other non-CuBLAS alerts: `NOTE: Before modifying/copying, see aten/src/ATen/Contex.h:alertNotDeterministic`

Reply:
As usual Ed and I appear to have the same idea. Your idea sounds good but I think you can just refer to the note's name (if you like the note approach).

Reply:
Ah, I see now that this "Note [...]" convention is used often throughout PyTorch, so I agree that this is the best way.